### PR TITLE
Update SRG-OS-000394-GPOS-00174 for RHEL 9 STIG

### PR DESCRIPTION
**Pull request by jan-cerny**
#### Description:

Add fixtext to rules

#### Rationale:
RHEL 9 STIG

### CODE REVIEW

File: linux_os/guide/system/software/integrity/crypto/configure_crypto_policy/rule.yml [hunk 1]
```diff
@@ -107,3 +107,9 @@ warnings:
         party review by an accredited lab. While open source software is
         capable of meeting this, it does not meet FIPS-140 unless the vendor
         submits to this process.
+
+fixtext: |-
+    Configure {{{ full_name }}} to use system cryptography policy.
+    Run the following command:
+
+    $ sudo update-crypto-policies --set {{{ xccdf_value("var_system_crypto_policy") }}}
```
Comment (Mab879) [inline linux_os/guide/system/software/integrity/crypto/configure_crypto_policy/rule.yml:115]: Please set `var_system_crypto_policy` to `fips` in the control.

File: linux_os/guide/system/software/integrity/crypto/harden_ssh_client_crypto_policy/rule.yml [hunk 2]
```diff
@@ -36,3 +37,16 @@ ocil: |-
...
+
+fixtext: |-
+    Configure the SSH client to override the system cryptographic policy.
+
+    Create file "/etc/ssh/ssh_config.d/02-ospp.conf" with the following content:
+
+    Match final all
+    RekeyLimit 512M 1h
+    GSSAPIAuthentication no
+    Ciphers aes256-ctr,aes256-cbc,aes128-ctr,aes128-cbc
```
Comment (Mab879) [inline linux_os/guide/system/software/integrity/crypto/harden_ssh_client_crypto_policy/rule.yml:49]: I think we will need some more review on this. I don't we think can in good faith recommend AES128-CBC in 2022. We might have to talk to the crypto team to get more details.

**Review by Mab879 — COMMENTED** [review]
One change and then starting the discussion on crypto algorithms.

**Comment by jan-cerny** [comment]
I have set the variable to fips.

File: linux_os/guide/system/software/integrity/crypto/harden_ssh_client_crypto_policy/rule.yml (hunk 2 above)
Comment (jan-cerny) [reply linux_os/guide/system/software/integrity/crypto/harden_ssh_client_crypto_policy/rule.yml:49]: Yes, good catch!

FTR it comes from https://github.com/ComplianceAsCode/content/pull/4681 which has been done for Common Criteria.

**Comment by jan-cerny** [comment]
I have resolved the conflicts and rebased it on the top of the latest upstream master branch.

File: controls/srg_gpos/SRG-OS-000394-GPOS-00174.yml [hunk 3]
```diff
@@ -9,4 +9,5 @@ controls:
             - configure_crypto_policy
             - harden_ssh_client_crypto_policy
```
Comment (Mab879) [inline controls/srg_gpos/SRG-OS-000394-GPOS-00174.yml:10]: This rule does not seem to be in another STIG, lets remove it.

File: linux_os/guide/system/software/integrity/crypto/harden_ssh_client_crypto_policy/rule.yml (hunk 2 above)
Comment (Mab879) [reply linux_os/guide/system/software/integrity/crypto/harden_ssh_client_crypto_policy/rule.yml:49]: We should just remove this rule. Comment no longer revlant.

**Review by Mab879 — CHANGES_REQUESTED** [review]

**Review by Mab879 — APPROVED** [review]

**Pull request merged**